language/stdlib/modules/VASP.move

```diff
@@ -164,6 +164,15 @@ module VASP {
         *&borrow_global<ParentVASP>(parent_address(addr)).num_children
     }
 
+    /// Return whether the VASP at `addr` is expired based upon the parent
+    /// VASP's `expiration_date` field.
+    public fun is_expired(account_address: address): bool
```

Comment:
Should we make this return `false` for any non-VASP account, or does an abort make sense?

Reply:
Good question! My thinking was abort since other functions in the module will abort if they're called on a non-VASP account. I think it will make it easier to have it return false if we move the check for VASPs above the if. I'll update :)